src/scala/microsoft-spark-2-4/src/main/scala/org/apache/spark/api/dotnet/CodegenUtils.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.apache.spark.api.dotnet
+
+import org.apache.spark.ml._
+import scala.collection.JavaConverters._
+
+/** DotnetHelper object that hosts some helper functions
+  * help data type conversions between dotnet and scala
+  */
+object DotnetHelper {
+
+  /** A helper function to let pipeline accept java.util.ArrayList
+    * format stages in scala code
+    * @param pipeline - The pipeline to be set stages
+    * @param value - A java.util.ArrayList of PipelineStages to be set as stages
+    * @return The pipeline
+    */
+  def setPipelineStages(pipeline: Pipeline, value: java.util.ArrayList[_ <: PipelineStage]): Pipeline =
```

Comment:
Should we move this method out into its own file/package?

Maybe package should be `org.apache.spark.ml.api.dotnet` and put in the file `MLUtils` and enclosed within the `object MLUtils`